Cargo.toml

```diff
@@ -47,7 +47,7 @@ simd = ["simd_backend"]
 avx2 = ["simd_backend"] # deprecated alias for simd_backend
 wasm = ["wasm-bindgen", "getrandom/js"]
 ffi = []
-no_cc_nightly = ["clear_on_drop/nightly"]
+no_cc_nightly = []
```

Comment:
This feature exists to compile clear_on_drop for envs like wasm which struggle when needing a C compiler at times.
Was the  `nightly` feature removed from the `clear_on_drop` crate?

Reply:
No I think it’s failing to compile with that feature on this nightly 

https://github.com/tari-project/tari-crypto/runs/4854291267?check_suite_focus=true

Reply:
yup, llvm_asm! support was dropped in rust and this feature uses it. We want to replace clear_on_drop with `zeroize` since clear_on_drop doesn't seem actively maintained.

Reply:
Zeroize is also used in the underlying dalek crate

Reply:
#75 has been merged